### PR TITLE
Add release notes for 1.6.0 release

### LINKED ISSUE #82: Add release notes for 1.6.0 release

### PR DESCRIPTION
**Pull request by korvoj**
@llopat please let me know if I missed out some feature.

### CODE REVIEW

File: docs/release-notes/1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Once the global administrator has indicated that a manual update has been performed to a new version, the nmaas platform will continue tracking subsequent update paths from that point forward.
+
+The manual update functionality is accessible via the `Actions -> Manual version update` menu of a particular application instance.
+
+<figure markdown>
+  ![Manual Version Update](img/04-manual-version-update.png)
+  <figcaption>Manually updating the version of an application instance</figcaption>
+</figure>
+
+### Automatic Helm Repository Updates in the Background
```
Comment (llopat) [inline docs/release-notes/1.6.0.md]: This helm repo update execution improvement was actually released in 1.5.4 so I suppose we can add a dedicated page for 1.5.4 as well.

File: docs/release-notes/1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Until now, the nmaas Platform was executing the `helm repo update` command via the `nmaas-helm` component upon each application deployment. This slowed down the application instance deployment in cases of unresponsive Helm repositories or a large number of repositories that need to be refreshed. The nmaas chart, and by extension the nmaas Platform now exposes two additional configuration properties controlling this behavior: `platform.properties.helm.asyncUpdateEnabled` and `platform.properties.helm.asyncUpdateCron`. The latter accepts a cron-style expression for scheduling when `helm repo update` should be run in case `asyncUpdateEnabled ` is set to `true`. A `helm repo update` is unconditionally run upon each new application version addition and `nmaas-platform` startup. 
+
+## Virtual Lab
+
+### New Virtual Lab Manager Role
+
+A new system role has been added to the Platform component, `Virtual Lab Manager` that can assigned to people responsible for organizing virtual labs on nmaas, without giving them a global administrator role. Virtual lab managers are allowed to:
```
Comment (llopat) [inline docs/release-notes/1.6.0.md]: "that can **be** assigned"

File: docs/release-notes/1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<figure markdown>
+  ![Manual Version Update](img/04-manual-version-update.png)
+  <figcaption>Manually updating the version of an application instance</figcaption>
+</figure>
+
+### Automatic Helm Repository Updates in the Background
+
+Until now, the nmaas Platform was executing the `helm repo update` command via the `nmaas-helm` component upon each application deployment. This slowed down the application instance deployment in cases of unresponsive Helm repositories or a large number of repositories that need to be refreshed. The nmaas chart, and by extension the nmaas Platform now exposes two additional configuration properties controlling this behavior: `platform.properties.helm.asyncUpdateEnabled` and `platform.properties.helm.asyncUpdateCron`. The latter accepts a cron-style expression for scheduling when `helm repo update` should be run in case `asyncUpdateEnabled ` is set to `true`. A `helm repo update` is unconditionally run upon each new application version addition and `nmaas-platform` startup. 
+
+## Virtual Lab
```
Comment (llopat) [inline docs/release-notes/1.6.0.md]: Well, in 1.6.0 we have in fact introduced the entire Virtual Lab functionality set so this part needs to be enhanced.

File: mkdocs.yml [hunk 4]
```diff
@@ -69,6 +69,9 @@ nav:
     - NMaaS-101 - "I can help! But... Please could you open these two doors?": 'blog/blog2/nmaas-101-2.md'
     - NMaaS-101 - "My name is Oxidized and I'm a nifty configuration management tool": 'blog/blog3/nmaas-101-3.md'
     - NMaaS-101 - "Prometheus" : 'blog/blog4/nmaas-101-4.md'
+  - Release Notes: 
+    - '1.6.0': 'release-notes/1.6.0.md'
```
Comment (korvoj) [inline mkdocs.yml:73]: @llopat, what makes more sense for you, to have the most recent version on top or at the bottom?

File: docs/release-notes/1.6.0.md (hunk 3 above)
Comment (korvoj) [reply docs/release-notes/1.6.0.md]: Great, thanks, I have extended this section with additional information.

File: mkdocs.yml (hunk 4 above)
Comment (llopat) [reply mkdocs.yml:73]: right, I think on top would be indeed better

**Review by llopat — APPROVED** [review]

**Pull request merged**